injitest/src/main/java/inji/pages/OtpVerificationPage.java

```diff
@@ -44,10 +45,15 @@ public class OtpVerificationPage extends BasePage {
     @iOSXCUITFindBy(iOSClassChain = "**/XCUIElementTypeStaticText[`label == \"VID not available in database\"`]")
     private WebElement vidNotAvailableMessage;
 
-    @AndroidFindBy(uiAutomator = "new UiSelector().resourceId(\"resendCodeView\")") //as parent component has correct property 
+    @AndroidFindBy(xpath = "//android.view.ViewGroup[@resource-id=\"resendCodeView\"]") //as parent component has correct property
```

Comment:
as parent component has correct property => Not using accessibility id as parent component has correct element property